pages/cart.vue

```diff
@@ -88,6 +89,8 @@
 import { SfButton, SfImage, SfHeading, SfBreadcrumbs, SfInput } from "@storefront-ui/vue"
 import { useAsync } from "@nuxtjs/composition-api"
 import { defineComponent } from "@vue/composition-api"
+import { useNuxtApp } from "#app"
```

Comment:
useRouter to push the path change instead

Reply:
using useRouter() now.